css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  background-color: #1996da;
+}
+
+.container {
+  max-width: 950px;
+  margin: auto;
+}
+
+h1 {
+  font-family: Lato, sans-serif;
+  color: #fff;
+  margin-top: 100px;
+  margin-bottom: 50px;
+}
+
+h2 {
+  font-family: Lato, sans-serif;
+  color: #fff;
+  margin-top: 30px;
+  margin-bottom: 10px;
+  font-size: large;
+}
+
+p {
+  font-family: Lato, sans-serif;
+  color: #fff;
+  margin: 10px;
+  text-align: center;
+  padding: 5px 15px;
+}
+
+img {
+  height: 75px;
+  margin-top: 20px;
+  margin-left: 25px;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.column-third {
+  margin: auto;
+  flex-basis: calc(100% / 3);
+}
+
+.border {
+  justify-content: center;
+  align-items: center;
```

Comment:
This isn't a flexbox, so these aren't doing anything. If you add `display: flex` to this rule, they will have an effect, and you will be able to remove the margin from your `img` elements.